modules/http/src/main/java/org/mule/module/http/internal/listener/DefaultHttpListenerConfig.java

```diff
@@ -334,4 +327,8 @@ public void setConnectionIdleTimeout(int connectionIdleTimeout)
         this.connectionIdleTimeout = connectionIdleTimeout;
     }
 
+    public void setConnectionManager(HttpListenerConnectionManager connectionManager)
```

Comment:
This field was not supposed to be mutable, a constructor argument would be better than a setter